db/schema.rb

```diff
@@ -187,6 +187,7 @@
     t.integer "division_id"
     t.boolean "eligible", default: false
     t.integer "slots_available", default: 0
+    t.boolean "looking_for_members", default: true, null: false
```

Comment:
Changes to the schema file without any migrations is going to probably cause problems, this PR most likely needs to be rebased against the `mitre-cyber-academy/ctf-scoreboard` master branch.